src/lib/components/Connect/steps/LinkSafe.svelte

```diff
@@ -24,17 +24,18 @@
   let selectedSafeAddress: string;
 
   onMount(async () => {
-    const ownedSafes = await getSafesForAddress(
-      $walletStore.network.chainId,
-      $walletStore.address
-    );
+    const { network, address } = $walletStore;
+
+    if (!network || !address) throw new Error('Ensure a wallet is connected');
+
+    const ownedSafes = await getSafesForAddress(network.chainId, address);
 
     if (!ownedSafes || ownedSafes.length === 0) {
       return modal.hide();
     }
 
     safeAddresses = ownedSafes;
-    selectedSafeAddress = safeAddresses[0];
+    selectedSafeAddress = ownedSafes[0];
```

Comment:
Was this incorrect previously?

Reply:
nope, it was fine before — the problem is that safeAddresses is typed as `string[] | undefined` so now with strict mode I can no longer assign index 0 without checking that it's defined. But `ownedSafes` is guaranteed to be a `string[]` so selecting from that instead works fine. In reality both variables have the same exact value in this moment.